substrate/frame/contracts/src/tests.rs

```diff
@@ -3511,7 +3512,7 @@ fn failed_deposit_charge_should_roll_back_call() {
 	let result = execute().unwrap();
 
 	// Bump the deposit per byte to a high value to trigger a FundsUnavailable error.
-	DEPOSIT_PER_BYTE.with(|c| *c.borrow_mut() = ED);
+	DEPOSIT_PER_BYTE.with(|c| *c.borrow_mut() = 20);
```

Comment:
.rust fixtures are ~4x bigger we need to lower this value to trigger the desired error.

Reply:
ED was set to 200 here, which will make the deposit too expensive

Reply:
But if the deposit is to expensive it should still yield `FundsUnavailable`? Or does it run into another error then?

Reply:
yeah it fails "StorageDepositNotEnoughFunds"